filebeat/harvester/log.go

```diff
@@ -265,6 +267,6 @@ func (h *Harvester) close() {
 		h.file.Close()
 		logp.Debug("harvester", "Stopping harvester, closing file: %s", h.Path)
 	} else {
-		logp.Debug("harvester", "Stopping harvester, NOT closing file as file info not available: %s", h.Path)
+		logp.Warn("harvester", "Stopping harvester, NOT closing file as file info not available: %s", h.Path)
```

Comment:
what does this actually mean? Any chance we're leaking file descriptors (besides file being finally closed when GC'ed, but this GC doesn't count)?